xl/player/gst/engine.py

```diff
@@ -358,6 +358,7 @@ def __init__(self, engine):
         # again. Instead, wait for the user to try to play a track,
         # and maybe the issue has resolved itself (plugged device in?)
         self.needs_sink = True
+        self.current_sink = None
```

Comment:
I'm uncertain about adding another variable here. Why not just add a get_volume to the DynamicAudioSink?

Reply:
This would be the easiest thing, I guess. This is audio_sink, right?
But I am uncertain about your favorite way of solving.